tools/greenbone-nvt-sync.in

```diff
@@ -511,7 +512,7 @@ do_sync ()
       date > $OPENVAS_RUN_DIR/feed-update.lock
       sync_nvts
       echo -n $OPENVAS_RUN_DIR/feed-update.lock
-    )9>$OPENVAS_RUN_DIR/feed-update.lock
+    )9<$OPENVAS_RUN_DIR/feed-update.lock
```

Comment:
If the file doesn't exist, it will end with an error. It should be open with append flag.
```suggestion
    )9>>$OPENVAS_RUN_DIR/feed-update.lock
```
Then, if the flock is successful, the line 512 will overwrite the the content instead of append

Reply:
Okay, I've changed it in 5d6fa24ae981cbc89660da5cc9f3f78594b5e73b and also in the gvmd PR.